DEVELOPMENT.md

```diff
@@ -0,0 +1,96 @@
+
+##Aurora features
+Handlers for Aurora features are defined in `no.skatteetaten.aurora.boober.feature`.
+
+### Features
+Features must extend the interface [Feature](./src/main/kotlin/no/skatteetaten/aurora/boober/feature/Feature.kt).
+The interface include documentation for the functions that needs to be implemented.
+
+The implemented functions are used in [AuroraDeploymentContext](./src/main/kotlin/no/skatteetaten/aurora/boober/model/AuroraDeploymentContext.kt)
+used by [AuroraDeploymentContextService](./src/main/kotlin/no/skatteetaten/aurora/boober/service/AuroraDeploymentContextService.kt)
+which in turn is used by [DeployFacade](./src/main/kotlin/no/skatteetaten/aurora/boober/facade/DeployFacade.kt). 
+
+
+The interface provides two functions for generating resources: `generate` and `generateSequentially`.
+A feature will normally only need to implement the `generate` function which will be executed in parallel.
+The `generateSequentially` function is intended for resources where parallel generation may result in duplicate resources (for example shared databases [DatabaseFeature](./src/main/kotlin/no/skatteetaten/aurora/boober/feature/DatabaseFeature.kt)). 
+
+#### Handlers
+The handlers function returns a set of [AuroraConfigFieldHandler](./src/main/kotlin/no/skatteetaten/aurora/boober/model/AuroraConfigFieldHandler.kt)
+which provides handling of a given pointer with default value and validation.
```

Comment:
kanskje heller:
which provides handling of a given AuroraConfig field pointer

Reply:
endret